pkg/sinks/new_relic_infra.go

```diff
@@ -236,6 +236,23 @@ func disposeBody(response *http.Response) {
 func (ns *newRelicInfraSink) decorateEvent(flattenedEvent map[string]interface{}) {
 	flattenedEvent["eventRouterVersion"] = newRelicEventrouterVersion
 	flattenedEvent["clusterName"] = ns.clusterName
+
+	// we are setting lastTimestamp since if an events gets repeated the fistTimestamp and the creation timestamp is not modified
+	if t, ok := flattenedEvent["event.lastTimestamp"]; ok {
+		if s, ok := t.(string); ok {
+			t, err := time.Parse(time.RFC3339, s)
+			if err == nil {
+				flattenedEvent["timestamp"] = strconv.FormatInt(t.Unix()*1000, 10)
```

Comment:
Why are we multiplying by 1000? The agent code does not:
https://github.com/Kav91/infrastructure-agent/blob/65f4054f8212af4971e5b19249ffa8384eba2aac/internal/agent/plugin.go#L140

Reply:
I was doing it since in the backend is shown as ms.
I changed it back to s, but still is not being recorded